lightningd/opening_control.c

```diff
@@ -1202,6 +1203,184 @@ static struct command_result *json_fundchannel_start(struct command *cmd,
 	return command_still_pending(cmd);
 }
 
+static struct channel *stub_chan(struct command *cmd,
+								u64 id,
+								struct node_id nodeid, 
+								struct channel_id cid,
+								struct bitcoin_outpoint funding,
+								struct wireaddr_internal addr){
+	struct peer *peer;
+	peer = new_peer(cmd->ld,
+						0,
+						&nodeid,
+						&addr,
+						false
+						);
+	
+	struct lightningd *ld = cmd->ld;
+	
+
+	struct channel_config our_config;
+	memset(&our_config, 1, sizeof(struct channel_config));
```

Comment:
If we want to make sure we set every field we use, I prefer using dynamic allocation (i.e. `our_config = tal(tmpctx,...`) so valgrind will tell us (valgrind doesn't work well with stack variables).